storage/persistent_storage/persistent-storage-aws.adoc

```diff
@@ -76,12 +76,12 @@ include::modules/storage-persistent-storage-volume-format.adoc[leveloffset=+1]
 
 include::modules/storage-persistent-storage-aws-maximum-volumes.adoc[leveloffset=+1]
 
-For information about accessing additional storage options, such as volume snapshots, that are not possible with in-tree volume plug-ins, see xref:../../storage/container_storage_interface/persistent-storage-csi-ebs.adoc#persistent-storage-csi-ebs[AWS Elastic Block Store CSI Driver Operator].
+For information about accessing additional storage options, such as volume snapshots, that are not possible with in-tree volume plug-ins, see xref:../../storage/container_storage_interface/persistent-storage-csi-ebs.adoc#persistent-storage-csi-ebs[Amazon Elastic Block Store CSI Driver Operator].
 
 include::modules/storage-persistent-storage-volume-encrypt-with-kms-key.adoc[leveloffset=+1]
 
 [id="additional-resources_persistent-storage-aws"]
 [role="_additional-resources"]
 == Additional resources
 
-* See xref:../../storage/container_storage_interface/persistent-storage-csi-ebs.adoc#persistent-storage-csi-ebs[AWS Elastic Block Store CSI Driver Operator] for information about accessing additional storage options, such as volume snapshots, that are not possible with in-tree volume plugins.
+* See xref:../../storage/container_storage_interface/persistent-storage-csi-ebs.adoc#persistent-storage-csi-ebs[Amazon Elastic Block Store CSI Driver Operator] for information about accessing additional storage options, such as volume snapshots, that are not possible with in-tree volume plugins.
```

Comment:
I have updated on Line 87. Please let me know if this instance need to be revoked. Thank you!

Reply:
Revoked. Please confirm.